indico/modules/events/contributions/controllers/management.py

```diff
@@ -544,6 +544,26 @@ def _process(self):
         return jsonify_form(form)
 
 
+class RHManageContributionPublicationREST(RHManageContributionsBase):
+    """Manage contribution publication setting"""
+
+    def _process_GET(self):
+        published = contribution_settings.get(self.event, 'published')
+        return jsonify(published=published)
+
+    def _process_PUT(self):
+        contribution_settings.set(self.event, 'published', True)
+        self.event.log(EventLogRealm.management, EventLogKind.change, 'Contributions',
```

Comment:
I think we could use `EventLogKind.positive` here (and `negative` for unpublishing below)